pkg/protoform/kubeconfig.go

```diff
@@ -22,47 +22,38 @@ under the License.
 package protoform
 
 import (
-	"flag"
 	"os"
 	"path/filepath"
 
-	"github.com/juju/errors"
 	log "github.com/sirupsen/logrus"
 	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth" //for auths
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
-	"k8s.io/client-go/util/homedir"
 )
 
 // GetKubeConfig  will return the kube config
-func GetKubeConfig(kubeconfigpath string, insecureSkipTLSVerify bool) (*rest.Config, error) {
+func GetKubeConfig(kubeconfigpath string, insecureSkipTLSVerify bool, kubeConfigOnly bool) (*rest.Config, error) {
```

Comment:
Instead of adding a flag, why not separate the logic?  GetKubeConfig uses InClusterConfig if available, otherwise uses the existing logic in a new function.  sctl would just call the new function, thus skipping GetKubeConfig completely.  Cleaner impl than continually adding flags.

Reply:
updated